src/firefly/js/ui/CatalogSearchMethodType.jsx

```diff
@@ -252,4 +269,78 @@ var initRadiusArcSec = (max) => {
     } else {
         return parseFloat(1 / 3600).toString();
     }
-};
\ No newline at end of file
+};
+
+
+/**
+ * Helper to get polygon from image
+ */
```

Comment:
I don't think the helper functions defined below are used.  Also, they contain unused imports as well as undefine function reference.  If you want to keep these, it should be cleaned up and moved to a util type of file.